weldx/asdf/schemas/weldx.bam.de/weldx/core/graph/graph-1.0.0.yaml

```diff
@@ -0,0 +1,21 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/schemas/weldx/core/graph/graph-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/graph/graph-1.0.0"
+
+title: |
+  Graph edge.
+description: |
+  A directed graph.
```

Comment:
maybe the schema should also be named that way.

Reply:
it would make sense, maybe simply digraph? Not sure if we plan on using undirected graphs at some point but maybe..
In that case it would also make sense to rename or modify the edge schema and class, maybe remove `direction` from `required`

Reply:
i have refactored everything to be "directed", I think we can cover the undirected stuff later?